examples/picorv32_ram/picorv32_ram.py

```diff
@@ -0,0 +1,60 @@
+
+import importlib
+import os
+import siliconcompiler
+
+def build_top():
+    # Core settings.
+    design = 'picorv32_top'
+    target = 'skywater130_demo'
+    die_w = 1000
+    die_h = 1200
+
+    # Create Chip object.
+    chip = siliconcompiler.Chip(design)
+
+    # Set default Skywater130 PDK / standard cell lib / flow.
+    target_module = importlib.import_module(f'targets.{target}')
+    chip.use(target_module)
+
+    # Set design source files.
+    rootdir = os.path.dirname(__file__)
+    chip.input(os.path.join(rootdir, f"{design}.v"))
+    chip.input(os.path.join(rootdir, "picorv32.v"))
+    chip.input(os.path.join(rootdir, "sky130_sram_2k.bb.v"))
+    chip.input(os.path.join(rootdir, f"{design}.sdc"))
+
+    # Optional: Relax linting and/or silence each task's output in the terminal.
+    chip.set('option', 'relax', True)
+    chip.set('option', 'quiet', True)
+
+    # Set die outline and core area.
+    chip.set('constraint', 'outline', [(0,0), (die_w, die_h)])
+    chip.set('constraint', 'corearea', [(10,10), (die_w-10, die_h-10)])
+
+    # Setup SRAM macro library.
+    from sram import sky130_sram_2k
+    chip.use(sky130_sram_2k)
+    chip.add('asic', 'macrolib', 'sky130_sram_2k')
+
+    # Increase macro place halo to reduce global routing congestion on SRAM macro.
+    chip.set('tool', 'openroad', 'task', 'route', 'var', 'route', '0', 'grt_allow_congestion', 'true')
```

Comment:
Is this necessary? In my experience for Sky130 OpenROAD flow, even with multiple SRAM blocks and 50%+ utilization, GRT should come out clean

Reply:
It appears to be, although the utilization is only 34%.

Reply:
Yeah, Wenting took a look at the congestion report and gave me some suggestions for fixing the global routing.